tests/v2/validation/snapshot/snapshot.go

```diff
@@ -49,6 +50,13 @@ const (
 	WorkloadNamePostBackup       = "wload-after-backup"
 )
 
+type snapshotStruct struct {
```

Comment:
supernit; why leave struct in the name? could it be snapshotConfig or something like that instead?

actually, why do we need this struct at all, couldn't you just read into `etcdsnapshot.Config` instead of making this struct?

Reply:
I mean we talked of specifically having a struct here. This struct was made to not overbloat the number of strings that is being returned in the snapshot functions below. `etcdsnapshot.Config` is completely unrelated.